docs/search/get-started-with-search/search-page/histogram.md

```diff
@@ -0,0 +1,57 @@
+---
+id: histogram
+title: Using the Log Search Histogram
+sidebar_label: Histogram
+description: You can highlight a time range in the histogram for your search results to filter the search results based on that time range.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+## Change the Time Range in the Histogram
+
+You can highlight a time range in the search results histogram to filter your search results in the **Messages** tab based on that time range.  
+
+* Your search results must be complete or paused for this feature to work.
+* Only the **Messages** tab is filtered. The **Aggregates** tab doesn't change.
+
+Click a bar in the histogram and use your cursor to select a contiguous set of bars. The search results update automatically to show only the results for the selected time range. The overall settings on the page don't change, but the message list is filtered to show only the messages for the selected period.
+
+![histogram drilldown](/img/search/get-started-search/search-page/histogram-drilldown.png)
+
+* Press the **X** in the selection area to clear a selection.
+* Click elsewhere in the histogram to make another selection.
+* Press **Shift Click** in the selected area (pink highlighted area) to open a new search tab for the selected time range.
+* Your selection is maintained if you go to another tab, but not if you close or save the search.
+
+If you'd prefer not to display the Log Search histogram, click **Hide Histogram**.
+
+<img src={useBaseUrl('img/search/get-started-search/search-page/hide-histogram.png')} alt="Hide Histogram" width="300"/>
+
+## Filter by Log Level
+
+When performing ad-hoc **Log Search** queries, you can visualize and filter log-level distribution in both your Histogram results and Messages table view, helping you to:
+
+* Quickly identify anomalies
+* Drill down in the high severity logs quickly
+* Navigate through a large volume of logs
+* Filter the relevant logs in their troubleshooting workflows
+
+<!--
+<details><summary>What are log levels?</summary>
+
+define warn, critical, etc.
+
+</details>
+-->
```

Comment:
@tarunk2 I'm thinking we should briefly define Sumo Logic log levels? I realized it's not documented anywhere.

Reply:
Makes sense. We detect 6 log levels out of the box: FATAL, ERROR, WARN, INFO, DEBUG and TRACE. If we are not able to find one of these log levels in a message, it is categorized under "OTHERS" bucket.